### PR TITLE
Fixed buf_fill_pct and gop_size in h264_settings

### PR DESCRIPTION
**Pull request by jonasdkhansen**
### Description
Fixed attributes in h264_settings:
buf_fill_pct
gop_size

### Relations

Relates #27824

### Output from Acceptance Testing

```
make testacc TESTS=TestAccMediaLiveChannel_videoDescriptions_codecSettings_h264Settings PKG=medialive
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./internal/service/medialive/... -v -count 1 -parallel 20 -run='TestAccMediaLiveChannel_videoDescriptions_codecSettings_h264Settings'  -timeout 180m
=== RUN   TestAccMediaLiveChannel_videoDescriptions_codecSettings_h264Settings
=== PAUSE TestAccMediaLiveChannel_videoDescriptions_codecSettings_h264Settings
=== CONT  TestAccMediaLiveChannel_videoDescriptions_codecSettings_h264Settings
--- PASS: TestAccMediaLiveChannel_videoDescriptions_codecSettings_h264Settings (85.31s)
PASS
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/medialive	107.335s
```

### CODE REVIEW

**Comment by jonasdkhansen** [comment]
> Hey @jonasdkhansen 👋 Thank you very much for your contribution! At times, our maintainers need to make direct edits to pull requests in order to help get it ready to be merged. Your current settings do not allow maintainers to make such edits. To help facilitate this, update your pull request to allow such edits as described in GitHub's [Allowing changes to a pull request branch created from a fork](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork) documentation. (If you're using a fork owned by an organization, your organization may not allow you to change this setting. If that is the case, let us know.)

We are aware of this, we are unable to change this at the current moment.

File: .changelog/30063.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+```release-note:bug
+resource/aws_medialive_channel: Fixed video_description and h264_settings for medialive
```
Comment (johnsonaj) [inline .changelog/30063.txt:2]:
```suggestion
resource/aws_medialive_channel: Fix type casting for `h264_settings` in `video_descriptions`
```

**Review by johnsonaj — COMMENTED** [review]
@jonasdkhansen over the PR looks good! I made one suggestion for the CHANGELOG entry.

Also, can you rebase/merge `main` into this PR?

**Review by johnsonaj — APPROVED** [review]
LGTM 🚀 

```console
$ make testacc TESTS=TestAccMediaLiveChannel_ PKG=medialive

==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./internal/service/medialive/... -v -count 1 -parallel 20 -run='TestAccMediaLiveChannel_'  -timeout 180m
--- PASS: TestAccMediaLiveChannel_audioDescriptions_codecSettings (71.33s)
--- PASS: TestAccMediaLiveChannel_disappears (72.97s)
--- PASS: TestAccMediaLiveChannel_UDP_outputSettings (87.63s)
--- PASS: TestAccMediaLiveChannel_hls (90.91s)
--- PASS: TestAccMediaLiveChannel_m2ts_settings (96.48s)
--- PASS: TestAccMediaLiveChannel_MsSmooth_outputSettings (148.06s)
--- PASS: TestAccMediaLiveChannel_videoDescriptions_codecSettings_h264Settings (149.47s)
--- PASS: TestAccMediaLiveChannel_basic (149.98s)
--- PASS: TestAccMediaLiveChannel_update (186.05s)
--- PASS: TestAccMediaLiveChannel_updateTags (192.49s)
--- PASS: TestAccMediaLiveChannel_status (342.48s)
PASS
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/medialive	345.699s
```

**Comment by johnsonaj** [comment]
@jonasdkhansen thanks for the contribution! 🎉

**Pull request merged**